src/store/main.js

```diff
@@ -62,6 +62,8 @@ export default new Vuex.Store({
 		navShown: true,
 		compactMode: localStorage.getItem('deck.compactMode') === 'true',
 		cardDetailsInModal: localStorage.getItem('deck.cardDetailsInModal') === 'true',
+		lastBoardId: localStorage.getItem('deck.lastBoardId'),
```

Comment:
Do we really need a global state for this? This could be easily stored in the modal component itself.